diff --git a/PKGBUILD b/PKGBUILD
index da896ea..e386e52 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -2,7 +2,7 @@
 # Contributor: Benjamin Klettbach <b.klettbach@gmail.com>
 # Contributor: Maciek Marciniak <mm2pl at kotmisia.pl>
 
-pkgname=obs-studio
+pkgname=obs-studio-system-theme
 pkgver=31.0.4
 pkgrel=1
 pkgdesc="Free, open source software for live streaming and recording"
@@ -22,11 +22,13 @@ optdepends=('libfdk-aac: FDK AAC codec support'
             'sndio: Sndio input client'
             'qrcodegencpp-cmake: websocket support'
             'v4l2loopback-dkms: virtual camera support')
-source=($pkgname-$pkgver.tar.gz::https://github.com/obsproject/obs-studio/releases/download/$pkgver/OBS-Studio-$pkgver-Sources.tar.gz)
-sha256sums=('ff9b49dbb89248064e4c9e9f8fb41dc8998bfa9d36842aae0a442a55f166ec0c')
+source=($pkgname-$pkgver.tar.gz::https://github.com/obsproject/obs-studio/releases/download/$pkgver/OBS-Studio-$pkgver-Sources.tar.gz use-system-theme-as-base.patch)
+sha256sums=('ff9b49dbb89248064e4c9e9f8fb41dc8998bfa9d36842aae0a442a55f166ec0c' SKIP)
 
 prepare() {
   cd $pkgname-$pkgver-sources
+
+  patch -p1 -i ../use-system-theme-as-base.patch
 }
 
 build() {
